pkg/network/protocols/testutil/serverutils.go

```diff
@@ -57,7 +57,7 @@ func RunDockerServer(t *testing.T, serverName, dockerPath string, env []string,
 	}
 }
 
-func RunHostServer(t *testing.T, command []string, env []string, serverStartRegex *regexp.Regexp) {
+func RunHostServer(t *testing.T, command []string, env []string, serverStartRegex *regexp.Regexp) bool {
```

Comment:
why do we see that change? it is already on `main` branch https://github.com/DataDog/datadog-agent/blob/main/pkg/network/protocols/testutil/serverutils.go#L27

Reply:
I rebased it

Reply:
this is RunHostServer()